discord/ext/commands/cooldowns.py

```diff
@@ -1,391 +1 @@
-"""
```

Comment:
i don't think we should move cooldowns  tbh

Reply:
> i don't think we should move cooldowns tbh


it's the best solution i saw to avoid circular imports. `discord.ext.commands.cooldowns` still importable. It's more like copying than moving

Reply:
Why not just copy cooldowns into discord.Bot?

Reply:
I'll try to move them to general